bundles/org.openhab.ui.homebuilder/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<parent>
+	    <groupId>org.openhab.core</groupId>
+    	<artifactId>pom-bundles</artifactId>
```

Comment:
some tab and space confusion in the pom 😄

Reply:
My favorite comment for such a case: https://www.emacswiki.org/pics/static/TabsSpacesBoth.png

Reply:
> My favorite comment for such a case: https://www.emacswiki.org/pics/static/TabsSpacesBoth.png

:+1: :rofl: